assets/js/handlebars/templates/settings-modal-body.handlebars

```diff
@@ -30,7 +30,7 @@
         <div class="switch-button__bg"></div>
       </div>
     </label>
-    <input class="input" type="url" name="livebook_url" placeholder="Type here the Livebook URL" />
+    <input class="input" type="url" name="livebook_url" placeholder="Type the Livebook URL here" />
```

Comment:
This might be more appropriate/simple/to the point

```suggestion
    <input class="input" type="url" name="livebook_url" placeholder="Enter Livebook URL" />
```

Another thing we can do is:
```suggestion
    <input class="input" type="url" name="livebook_url" placeholder="eg: https://livebook.dev" />
```